### DISCUSSION TITLE
What to backup?

### DISCUSSION THREAD

**Discussion by blogdemoi** [jonaswinkler/paperless-ng#1646] [Support] [answered]
Hello,

I just installed Paperless-ng on my Manjaro laptop (I am not interested in the server part to access paperless-ng remotely, I save in paperless-ng locally only). What files and folders do I need to backup if a something bad happens? (I use borg to backup to a remote server).

Thank you.

**Comment by ma651851384** [comment, marked as answer]
Hello @blogdemoi ,
you can find the answer of your question in the documentation of paperless-ng. Here is the link:
[readthedocs.io](https://paperless-ng.readthedocs.io/en/latest/administration.html)

I personally use the docker installation on an raspberrypi 3B and make use of the [file naming feature](https://paperless-ng.readthedocs.io/en/latest/advanced_usage.html#file-name-handling). That means the files are located in the storage of the Raspi with the prefered file/folder structure, I could live well, if paperless-ng would not work any more from one minute to the other. Of cause with that method, the database would not be part of my "backup"/sync.

In my case I clone the folder by rclone every 5 minutes to my Nextcloud instance. With that, I have all needed files available on every computer.

_You_ could define the /media folder to be on your home directory like /home/<username>/Documents/, if you use the docker installation. And you could also use borg.

I hope, that gives you an idea, how to process in your case.

Regards

**Reply by blogdemoi** [reply]
Thank you!

**Reply by blogdemoi** [reply]
Hello,

So, if I backup  /var/lib/paperless I backup all of my data and if I need to reinstall I just need to paste back this ?

Thank you.

**Reply by ma651851384** [reply]
You did not clearly descripe, how you have installed paperless-ng. Do you use docker?

**Reply by blogdemoi** [reply]
Sorry. I installed it following this method :

Install Paperless from Docker Hub using the installation script
`curl -L https://raw.githubusercontent.com/jonaswinkler/paperless-ng/master/install-paperless-ng.sh | bash
`

It is working fine, I am currently importing documents, tagging them, etc. I am just a little confused about which folders I need to backup in case something bad happens and if I need to reinstall.

**Reply by ma651851384** [reply]
Ok, thx for the information. The script can be [viewed](https://raw.githubusercontent.com/jonaswinkler/paperless-ng/master/install-paperless-ng.sh).

So, during installation, you were asked to insert four folder paths and which database you would like to use (sqlite or postget). Depending on your personal choosen settings during your individual installation, the folders were created. You should backup those folders. As the user can choose the installation paths indivitually, there is no general answer to your question.

There should be a "docker-compose.yml" file in your paperless-ng folder. Search there for the lines, which includes follow two strings: "/usr/src/paperless/media", "/usr/src/paperless/data".

In before the strings, there is a ":" (colon) and before the colon you can see the folder paths. Backup that folders, then you should be fine.
